src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -39,4 +41,22 @@
         public final static int XboxAbutton = 1;
         public final static int XboxXbutton = 3;
     }
+
+    public static class Encoders {
+        //changes Encoder ports into integers
+        public final static int LIFT_A = 1;
+        public final static int LIFT_B = 2;
+    }
+
+    public static class Solenoids {
+        //changes Solenoid ports into integers
+        public final static int LiftTransmission = 1;
+        public final static int LiftTransmission2 = 2;
```

Comment:
These should follow the same naming scheme LIFT_TRANSMISSION_A, LIFT_TRANSMISSION_B